mobify/progressive-web-scaffold
# [SCSS] Add missing SCSS variables that's used by SDK base and theme s…

Henry and Loki discovered an issue when they tried adding a SDK component (Banner) into Merlins. The CSS wasn't compiling because `$sub-unit` was used by Banner's base styles and we don't have `$sub-unit` defined in the scaffold's variables.

## Changes
- Add missing SCSS variables that's used by SDK component's base and theme styles into the scaffold's variables.
- Replaced the `$light-brand-color` and `$dark-brand-color` with `primary`, `secondary`, etc to match what's used in the SDK styles.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- The button's active/focus states should remain the same as before. Everything else should be untouched.
